tests/integration/sites_test.py

```diff
@@ -71,60 +71,24 @@ def test_transitions_parts(self, vasp_sites, vasp_transitions):
             np.array([[0, 0, 95], [4, 23, 95], [8, 28, 95]]))
 
     def test_occupancy(self, vasp_transitions):
-        occupancy = vasp_transitions.occupancy()
-        assert len(occupancy) == 95
-        assert sum(occupancy.values()) == 137026
-        assert list(occupancy.values())[::20] == [1704, 971, 351, 1508, 1104]
-
-    def test_occupancy_parts(self, vasp_sites, vasp_transitions):
-        parts = vasp_sites.occupancy_parts(vasp_transitions)
-        assert len(parts) == self.n_parts
-        assert [sum(part.values()) for part in parts] == [
-            13592, 13898, 13819, 14028, 14022, 14470, 13200, 13419, 13286,
-            13292
-        ]
-
-    def test_site_occupancy(self, vasp_sites, vasp_transitions):
-        assert isclose(vasp_sites.site_occupancy(vasp_transitions)['48h'],
-                       0.380628,
-                       rel_tol=1e-4)
+        structure = vasp_transitions.occupancy()
 
-    def test_site_occupancy_parts(self, vasp_sites, vasp_transitions):
-        assert len(
-            vasp_sites.site_occupancy_parts(vasp_transitions)) == self.n_parts
-        assert isclose(
-            vasp_sites.site_occupancy_parts(vasp_transitions)[0]['48h'],
-            0.37756,
-            rel_tol=1e-4)
-        assert isclose(
-            vasp_sites.site_occupancy_parts(vasp_transitions)[9]['48h'],
-            0.36922,
-            rel_tol=1e-4)
+        assert len(structure) == 96
+        assert structure[0].species.num_atoms == 0.4544
+        assert structure.composition.num_atoms == 37.54026666666666
 
     def test_atom_locations(self, vasp_sites, vasp_transitions):
         assert isclose(vasp_sites.atom_locations(vasp_transitions)['48h'],
-                       0.761255,
+                       0.7820889,
                        rel_tol=1e-4)
 
-    def test_atom_locations_parts(self, vasp_sites, vasp_transitions):
-        assert len(
-            vasp_sites.atom_locations_parts(vasp_transitions)) == self.n_parts
-        assert isclose(
-            vasp_sites.atom_locations_parts(vasp_transitions)[0]['48h'],
-            0.755111,
-            rel_tol=1e-4)
-        assert isclose(
-            vasp_sites.atom_locations_parts(vasp_transitions)[9]['48h'],
-            0.738444,
-            rel_tol=1e-4)
-
```

Comment:
We should test the atom_locations on a splitted transitions object instead of deleting the test